modules/sharedaddy/sharing-service.php

```diff
@@ -73,6 +73,10 @@ public function get_all_services( $include_custom = true ) {
 			$services['email'] = 'Share_Email';
 		}
 
+		if ( is_multisite() && ( version_compare( $wp_version, '4.9', '<' ) || is_plugin_active( 'press-this/press-this-plugin.php' ) ) ) {
```

Comment:
`version_compare` returns true here even if I'm on `4.9-RCX-XXXX`, which I'm assuming is incorrect. This is a minor thing, but I guess it should be brought in sync with the other version check in this PR.

Reply:
Updated in 3a4cfa6